store_test.go

```diff
@@ -272,6 +281,31 @@ func write(t *testing.T, ctx context.Context, store *Store, key string, value in
 	}
 }
 
+func testSessionSuccess(t *testing.T, ctx context.Context, store *Store) {
+	key := "foo/lock"
+	lock, unlock := TryLockOne(ctx, path.Join(store.Keyspace, key))
+	defer unlock()
+
+	if err := lock.Err(); err != nil {
+		t.Fatal(err)
```

Comment:
Can you replace the use of `Fatal` with `Error` here? It's always useful to give the developers as much info as possible.